cmd/integration-cip-gbg-karta/main.go

```diff
@@ -25,18 +29,37 @@ func main() {
 
 	cb := application.NewContextBrokerClient(contextBrokerUrl)
 
-	beaches, err := cb.GetBeaches(ctx)
+	flag.StringVar(&bcSelector, "bcSelector", "greenspacerecord", "Flag to distinguish which funcion to be selected for its business case")
+	conn, err := pgx.Connect(ctx, pgConnUrl)
 	if err != nil {
-		logger.Fatal().Err(err).Msg("unable to fetch beaches")
+		logger.Fatal().Err(err).Msg("unable to connect to database")
 	}
+	//	var conn *pgx.Conn
 
-	logger.Info().Msgf("fetched %d beaches from %s", len(beaches), contextBrokerUrl)
+	defer conn.Close(ctx)
 
-	conn, err := pgx.Connect(ctx, pgConnUrl)
+	if bcSelector == "beach" {
+		err := bcWaterQualityObserved(ctx, cb, logger, contextBrokerUrl, *conn)
+		if err != nil {
+			logger.Error().Err(err).Msg("error in bcWaterQualityObserved")
+		}
+	} else if bcSelector == "greenspacerecord" {
+		err := bcGreenspaceRecord(ctx, cb, logger, contextBrokerUrl, *conn)
+		if err != nil {
+			logger.Error().Err(err).Msg("error in bcWaterQualityObserved")
+		}
+	}
```

Comment:
... else we should output an error to notify the user that the configuration is wrong. This silently fails if you misspel the bc name.

Reply:
Use Fatal here to clearly signal that we are unable to do anything with the supplied configuration. Also, "no configuration found" implies an internal error, while "%s is not a supported business case" is more clear that the supplied configuration is wrong.